manifests/m/Microsoft/Edge/Canary/112.0.1705.0/Microsoft.Edge.Canary.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Edge.Canary
+PackageVersion: 112.0.1705.0
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://c2rsetup.edog.officeapps.live.com/c2r/downloadEdge.aspx?platform=Default&source=EdgeInsiderPage&Channel=Canary&language=en

```

Comment:
@denelon - Can you validate this is an official URL? It doesn't seem to match `Microsoft.Edge`, `Microsoft.Edge.Beta`, or `Microsoft.Edge.Dev`